curvefs/src/metaserver/storage/rocksdb_storage.cpp

```diff
@@ -30,171 +29,63 @@
-    cfOptions.level_compaction_dynamic_level_bytes = true;
-    cfOptions.compaction_pri = ROCKSDB_NAMESPACE::kMinOverlappingRatio;
-    // use the specified function to determine the prefixes for keys
-    cfOptions.prefix_extractor.reset(NewFixedPrefixTransform(sizeof(size_t)));
-    // The size in bytes of the filter for memtable is
-    // write_buffer_size * memtable_prefix_bloom_size_ratio
-    cfOptions.memtable_prefix_bloom_size_ratio =
-        options.memtablePrefixBloomSizeRatio;
-    // reset table options for column failmy
-    cfOptions.table_factory.reset(NewBlockBasedTableFactory(tableOptions));
-
-    ColumnFamilyOptions unorderedCFOptions = cfOptions;
-    // amount of data to build up in memory (backed by an unsorted log
-    // on disk) before converting to a sorted on-disk file.
-    unorderedCFOptions.write_buffer_size = options.unorderedWriteBufferSize;
-    // the maximum number of write buffers that are built up in memory.
-    unorderedCFOptions.max_write_buffer_number =
-        options.unorderedMaxWriteBufferNumber;
-
-    ColumnFamilyOptions orderedCFOptions = cfOptions;
-    orderedCFOptions.write_buffer_size = options.orderedWriteBufferSize;
-    orderedCFOptions.max_write_buffer_number =
-        options.orderedMaxWriteBufferNumber;
-
-    columnFamilies_.push_back(ColumnFamilyDescriptor(
-        ROCKSDB_NAMESPACE::kDefaultColumnFamilyName, unorderedCFOptions));
-    columnFamilies_.push_back(ColumnFamilyDescriptor(
-        kOrderedColumnFamilyName_, orderedCFOptions));
-}
-
-inline std::vector<ColumnFamilyDescriptor>
-RocksDBOptions::ColumnFamilys() {
-    return columnFamilies_;
-}
-
-inline ROCKSDB_NAMESPACE::Options RocksDBOptions::DBOptions() {
-    return dbOptions_;
-}
-
-inline ROCKSDB_NAMESPACE::TransactionDBOptions
-    RocksDBOptions::TransactionDBOptions() {
-    return ROCKSDB_NAMESPACE::TransactionDBOptions();
-}
-
-inline ROCKSDB_NAMESPACE::ReadOptions RocksDBOptions::ReadOptions() {
-    return ROCKSDB_NAMESPACE::ReadOptions();
-}
-
-inline ROCKSDB_NAMESPACE::WriteOptions RocksDBOptions::WriteOptions() {
-    auto options = ROCKSDB_NAMESPACE::WriteOptions();
-    options.disableWAL = true;
-    return options;
-}
-
 RocksDBStorage::RocksDBStorage()
-    : InTransaction_(false) {}
+    : InTransaction_(false) {
+    InitDbOptions();
+}
 
 RocksDBStorage::RocksDBStorage(StorageOptions options)
     : inited_(false),
-      options_(options),
-      rocksdbOptions_(RocksDBOptions(options)),
+      options_(std::move(options)),
       counter_(std::make_shared<Counter>()),
-      InTransaction_(false) {}
+      InTransaction_(false) {
+    InitDbOptions();
+}
 
 RocksDBStorage::RocksDBStorage(const RocksDBStorage& storage,
                                ROCKSDB_NAMESPACE::Transaction* txn)
     : inited_(storage.inited_),
       options_(storage.options_),
-      rocksdbOptions_(storage.rocksdbOptions_),
+    //   rocksdbOptions_(storage.rocksdbOptions_),
```

Comment:
why comment it?

Reply:
the RocksDBOptions is been deleted, and some optinos in it is moved into `RocksDBStorage`